src/netex-convertor/handler.ts

```diff
@@ -37,6 +37,19 @@ export const netexConvertorHandler = async (event: S3Event): Promise<void> => {
 
             const fileNameWithoutSlashes = fileName.replace('/', '_');
             await s3.uploadNetexToS3(generatedNetex, fileNameWithoutSlashes);
```

Comment:
Comparing the blocks of code we've got for type === 'pointToPoint' and the new type === 'return', could we have these two pieces of code as one? Is there any difference in the logic at this level?W We would then be making good use of the extended MatchingData type as well 😃

Reply:
Not sure if this bit was lost in the git mess we had?